package.json

```diff
@@ -67,6 +67,7 @@
     "remove-markdown": "^0.3.0",
     "rss": "^1.2.2",
     "source-map-explorer": "^2.5.2",
+    "sudo-prompt": "^9.2.1",
```

Comment:
This repository is archived on github. Something to keep an eye on, or fork for our selves.